### PR TITLE
Replace bigint(BigInt) to JSBI

### PR DESCRIPTION
**Pull request by MichaelKim20**
This PR is not necessary if all users use the latest version of the browser.
However, there are some policies to solve this problem, so I will prepare it before it is decided.

Related to #72

### CODE REVIEW

File: tests/ValidationJSON.test.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe ('Test that validation with JSON schema', () =>
                     "value": "400000000000",
                     "lock": {
                         "type": 0,
-                        "bytes": [156,198,57,161,53,47,119,242,37,22,12,66,255,137,26,106,250,91,70,161,26,105,169,73,32,134,215,120,6,118,244,42]
+                        "bytes": "WEjSPkXBh0iqzGnYinwWYyNckuHdOU3TNbVr7oURPBxFKyXaCXkRc0o3O2IwNZKple6"
```
Comment (TrustHenry) [inline tests/ValidationJSON.test.ts:100]: Is this related to this PR?

File: src/modules/common/Hash.ts [hunk 2]
```diff
@@ -154,10 +155,10 @@ export function hashMulti (source1: Buffer, source2: Buffer): Hash
  * @returns The instance of Hash
  * See_Also https://github.com/bpfkorea/agora/blob/93c31daa616e76011deee68a8645e1b86624ce3d/source/agora/consensus/data/UTXOSetValue.d#L50-L53
```
Comment (TrustHenry) [inline src/modules/common/Hash.ts:156]: https://github.com/bpfkorea/agora/blob/382c3a414878d26196acc3595a8198903e98e866/source/agora/consensus/data/UTXO.d#L46-L49

Comment (MichaelKim20) [reply src/modules/common/Hash.ts:156]: Yes, it's the same content.

File: tests/ValidationJSON.test.ts (hunk 1 above)
Comment (MichaelKim20) [reply tests/ValidationJSON.test.ts:100]: I'll post this on another PR.

**Review by TrustHenry — APPROVED** [review]
@Geod24 What do you think about this change?

File: src/modules/common/Height.ts [hunk 3]
```diff
@@ -22,15 +23,15 @@ export class Height
...
      * the block height
      */
-    public value: bigint;
+    public value: JSBI;
 
     /**
      * Construct
      * @param value The block height
      */
-    constructor (value: bigint | string)
```
Comment (Geod24) [inline src/modules/common/Height.ts:31]: Why remove BigInt ?

File: src/modules/data/TxOutput.ts [hunk 4]
```diff
@@ -40,9 +41,9 @@ export class TxOutput
      * @param value   The monetary value
      * @param lock    The public key or instance of Lock
      */
-    constructor (value: bigint, lock: Lock | PublicKey)
+    constructor (value: string, lock: Lock | PublicKey)
```
Comment (Geod24) [inline src/modules/data/TxOutput.ts:44]: That's pessimization. Try to keep it typed as bigint, instead of dropping to string

File: src/modules/net/response/UnspentTxOutput.ts [hunk 5]
```diff
@@ -59,8 +61,8 @@ export class UnspentTxOutput
      * @param height        The height of the block on created
      * @param time          The time of the block on created
      */
-    constructor (utxo?: Hash, type?: TxType, unlock_height?: bigint, amount?: bigint,
-                 height?: bigint, time?: number)
+    constructor (utxo?: Hash, type?: TxType, unlock_height?: string, amount?: string,
+                 height?: string, time?: number)
```
Comment (Geod24) [inline src/modules/net/response/UnspentTxOutput.ts:65]: Same thing about BigInt vs string

**Comment by Geod24** [comment]
> @Geod24 What do you think about this change?

We shouldn't replace `bigint` with `string`. The rest looks okay tho.

**Comment by MichaelKim20** [comment]
@Geod24 @TrustHenry I applied all the requests.

**Comment by MichaelKim20** [comment]
I rebased it

**Review by TrustHenry — APPROVED** [review]
